src/postamble.js

```diff
@@ -187,7 +187,7 @@ Module['callMain'] = function callMain(args) {
   HEAP32[(argv >> 2) + argc] = 0;
 
 #if EMTERPRETIFY_ASYNC
-  var initialEmtStackTop = Module['asm']['emtStackSave']();
+  var initialEmtStackTop = Module['emtStackSave']();
```

Comment:
Was this change intentional? (doesn't look just like a asm.js -> wasm defaults change?)

Reply:
It's a bugfix, that method is exported and should be accessed like any other exported method, on module - this was noticeable in wasm because of async compilation which changes when `Module['asm']` is set. Could have been in a separate PR though, now that I think of it, sorry about that.

Reply:
Or actually I think this was an issue with metadce which is only in wasm (and needs proper usages, not ones on Module.asm), and not async startup.